control/control.rnc

```diff
@@ -308,6 +309,7 @@ software_elements =
 ## Whether it is allowed to delete a package during upgrade
 ## *obsolete* without replacement
 delete_old_packages = element delete_old_packages { BOOLEAN }
+disable_media_repo = element disable_media_repo { BOOLEAN }
```

Comment:
I'd add a comment about this feature (with the FATE number) for a future reference.